### PR TITLE
data: Vishal21121

### PR DESCRIPTION
**Pull request by Vishal21121**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5758"></a>

### CODE REVIEW

File: data/Vishal21121.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Vishal Singh",
+  "displayStatsPublic": true,
```
Comment (yoyoekta) [inline data/Vishal21121.json:3]: Thank you for submitting this pull request. I noticed that the `avatar` and `displayStatsPublic` properties are no longer used. Would it be possible to remove it from the pull request to keep everything up to date? 
```suggestion
 
```

File: data/Vishal21121.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "type": "personal",
+  "bio": "A full stack developer and a student pursuing B.Tech",
+  "avatar": "https://github.com/Vishal21121.png",
```
Comment (yoyoekta) [inline data/Vishal21121.json:6]:
```suggestion
  
```

**Comment by yoyoekta** [comment]
I added some inline comments.

**Comment by Vishal21121** [comment]
Do I have to make a new pull request?

**Review by TRohit20 — DISMISSED** [review]

**Comment by SaraJaoude** [comment]
> Do I have to make a new pull request?

You do not need to make a new Pull Request but you can edit your file.  However I have marked the conversation as `unresolved` and committed these changes, to save you having to edit your file. 

If you want to make any changes to your Profile in the future, then raise a new Pull Request.  You might want to have a look at our Editing Guide in our Docs before you do this.  https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**